### PR TITLE
W-16199921 Updating Einstein Prereqs

### PR DESCRIPTION
**Pull request by valkyrie69**

### CODE REVIEW

File: modules/ROOT/pages/int-ai-enable-einstein.adoc [hunk 1]
```diff
@@ -40,23 +46,25 @@ If the connection is unsuccessful, it might mean that Einstein isn't enabled by
 +
 Einstein is now enabled for all the users in your Anypoint Platform organization. You can remove the *Mule Developer Generative AI User* permission to access Einstein from individual users in xref:access-management::users.adoc#removing-permissions-from-user[access management]. 
 
-== Change the Connected Salesforce Organization
+== Changing the Connected Salesforce Organization
```
Comment (bjoson-mulesoft) [inline modules/ROOT/pages/int-ai-enable-einstein.adoc]: Just curious why the change from imperative to gerund? I thought our standard was gerund for the topic name and imperative for the tasks.

Comment (valkyrie69) [reply modules/ROOT/pages/int-ai-enable-einstein.adoc]: @bjoson-mulesoft I always get this mixed up. haha

**Review by bjoson-mulesoft — APPROVED** [review]
